pallets/xcm-weight-trader/src/lib.rs

```diff
@@ -151,7 +175,7 @@ pub mod pallet {
 	#[pallet::call]
 	impl<T: Config> Pallet<T> {
 		#[pallet::call_index(0)]
-		#[pallet::weight(T::WeightInfo::add_asset())]
```

Comment:
Was this giving an error?

Reply:
Removed it. Seems like it was left dangling from a previous implementation.